configure.ac

```diff
@@ -748,6 +748,21 @@ AC_ARG_ENABLE(
   [AS_HELP_STRING([--disable-doxygen-version],
   [Substitute the Doxygen version with latest, for the website])])
 
+# systemd support
+AC_ARG_ENABLE(
+  [systemd],
+  [AS_HELP_STRING([--enable-systemd], [Enable systemd support])])
```

Comment:
You probably want to negate this, so it's turned on by default where available (or no-one will realise it exists/use it). See e.g. https://github.com/OpenLightingProject/ola/blob/master/configure.ac#L573-L589 and don't forget to update the AS_IF below too.

Reply:
Eek.. an automatic dependency feels a little weird to me, unless we put it out for people to know. If we put the existence of this option in `NEWS` (which is probably where it will end up in) and you suspect:
>  no-one will realise it exists/use it

then, if we put in a warning that `libsystemd` is going to become an automatic dependency, I kinda also think that
> no-one will realise it exists/use it

until bug reports come in regarding `libsystemd` not being found on non-build systems...

Reply:
Hmm, we have the best of both worlds with our plugins, a literal auto mode, so by default if the bits are there, you get your plugin and if they aren't you don't, but you can also force it on (so it complains if dependencies are missing) or force it off (so it doesn't use it even if present).

You may be right about bug reports, I guess I'd tend towards the nudge theory, if we turn it on, then people who read the docs will turn it off easily, those who don't will ideally search or file a bug/ask on the mailing list and there's a workaround.

If we leave it off, we should at least enable it in our Debian build config.

Reply:
Hmm, alright. Let's default it to enabled. We're going to be playing devil's advocate (to some people out there) and _silently_ promote `systemd` adoption...

Reply:
Fixed in commit https://github.com/OpenLightingProject/ola/pull/1444/commits/83807d484ad4d66cc447a28ef395019fa13ff906